OTKit/otkit-icons/ic_card_american_express.svg

```diff
@@ -0,0 +1,23 @@
+<svg width="40" height="24" viewBox="0 0 40 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <title>Card/American Express</title>
```

Comment:
Is there a specific format these title's should follow? I copied this format from the social buttons.

Reply:
@ricalebro No specific format. We use SVGO to clean up SVGs during the build phase